scipy/stats/_qmc.py

```diff
@@ -1183,13 +1236,13 @@ def __init__(self, mean, cov=None, cov_root=None, inv_transform=True,
         else:
             engine_dim = d
         if engine is None:
-            self.engine = Sobol(d=engine_dim, scramble=True, seed=seed)
+            self.engine = Sobol(d=engine_dim, scramble=True, seed=seed) # type: QMCEngine
```

Comment:
Had to add this to tel Mypy that `self.engine` has `QMCEngine` type. See scipy/stats/_qmc.py:1300: error: Module has no attribute "norm"  [attr-defined]